code/game/objects/items/melee/misc.dm

```diff
@@ -542,7 +542,7 @@
 /obj/item/melee/roastingstick/Initialize()
 	. = ..()
 	if (!ovens)
-		ovens = typecacheof(list(/obj/singularity, /obj/machinery/power/supermatter_crystal, /obj/structure/bonfire, /obj/structure/destructible/clockwork/massive/ratvar))
```

Comment:
single biggest lost.

Reply:
I may have made a mistake there.

Reply:
Actually no we good, it's good.

Reply:
Though if someone wants to change that to `/obj/structure/dead_ratvar` by all means.